libsolidity/ast/AST.cpp

```diff
@@ -333,7 +333,7 @@ FunctionTypePointer FunctionDefinition::functionType(bool _internal) const
 		case Visibility::Private:
 		case Visibility::Internal:
 		case Visibility::Public:
-			return TypeProvider::function(*this, _internal);
+			return TypeProvider::function(*this, FunctionType::Kind::Internal);
```

Comment:
Shouldn't this take `_internal` into account?

Reply:
Look at the enclosing ``if``.

Reply:
looked

Reply:
Satisfied then, I take it :-)?

Reply:
yes